easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '5.4.1'
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale
+materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics,
+from first principles."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+prebuildopts = 'cp arch/makefile.include.linux_intel ./makefile.include; '
+prebuildopts += 'env LIBS="" '
+
+buildopts = 'all BLACS="$LIBBLACS" '
+buildopts += 'OBJECTS="fftmpiw.o fftmpi_map.o fftw3d.o fft3dlib.o $FFTW_LIB_DIR/libfftw3xf_intel.a"'
```

Comment:
well, before including a comment, the issue is that the provided makefile.include has the path to libfftw3xf_intel.a hardcoded in the same variable as the other fft objects,

```
OBJECTS    = fftmpiw.o fftmpi_map.o fftw3d.o fft3dlib.o \
             $(MKLROOT)/interfaces/fftw3xf/libfftw3xf_intel.a
```

so I had to override the whole variable. Alternatively, I could have used sed on the makefile.include. Is there a better way?